### PR TITLE
stubsabot: Add the PR body to the commit message

### PR DESCRIPTION
**Pull request by AlexWaygood**
Having the PR body as part of the commit message makes it easier to test changes to the stubsabot PR messages if you're hacking on stubsabot locally. (As it currently is, it's difficult to test changes to the PR message unless you run stubsabot with `--action-level everything`, which will actually create a PR against typeshed.)

### CODE REVIEW

File: scripts/stubsabot.py [hunk 1]
```diff
@@ -348,6 +330,28 @@ async def suggest_typeshed_update(update: Update, session: aiohttp.ClientSession
...
+    async with _repo_lock:
+        branch_name = f"{BRANCH_PREFIX}/{normalize(update.distribution)}"
+        subprocess.check_call(["git", "checkout", "-B", branch_name, "origin/master"])
+        with open(update.stub_path / "METADATA.toml", "rb") as f:
+            meta = tomlkit.load(f)
+        meta["version"] = update.new_version_spec
+        with open(update.stub_path / "METADATA.toml", "w") as f:
+            tomlkit.dump(meta, f)
+        body = get_pr_body(update, meta)
+        subprocess.check_call(["git", "commit", "--all", "-m", f"{title}\n{body}"])
```
Comment (hauntsaninja) [inline scripts/stubsabot.py:349]: nit: double newline needed?

File: scripts/stubsabot.py [hunk 2]
```diff
@@ -308,28 +308,10 @@ def normalize(name: str) -> str:
...
-            return
-        somewhat_safe_force_push(branch_name)
-        if action_level <= ActionLevel.fork:
-            return
-
-    body = "\n".join(f"{k}: {v}" for k, v in update.links.items())
-
-    stubtest_will_run = not meta.get("stubtest", {}).get("skip", False)
+def get_pr_body(update: Update, metadata: dict[str, Any]) -> str:
+    body = "Project links:\n - "
```
Comment (hauntsaninja) [inline scripts/stubsabot.py:312]: nit: will have stray dash if there aren't update.links. the update.links are labelled and filtered to only changelog + release + home page, so not sure we need this

**Review by hauntsaninja — COMMENTED** [review]
Want to make a similar change for `suggest_typeshed_obsolete`?

**Comment by AlexWaygood** [comment]
> Want to make a similar change for `suggest_typeshed_obsolete`?

Oh, sure!

**Review by hauntsaninja — APPROVED** [review]

**Pull request merged**